src/base/bittorrent/session.cpp

```diff
@@ -4206,7 +4210,7 @@ void Session::handlePeerBlockedAlert(libt::peer_blocked_alert *p)
         reason = tr("because it has a low port.", "this peer was blocked because it has a low port.");
         break;
     case libt::peer_blocked_alert::utp_disabled:
-        reason = trUtf8("because %1 is disabled.", "this peer was blocked because uTP is disabled.").arg(QString::fromUtf8(C_UTP)); // don't translate μTP
+        reason = trUtf8("because %1 is disabled.", "this peer was blocked because uTP is disabled.").arg(QString::fromUtf8(C_UTP)); // don't translate ??TP
```

Comment:
please fix this, just use the `u` in ASCII

Reply:
Ok. Editor changed without notice.